tests/e2e/pageobjects/ide/DebugView.ts

```diff
@@ -41,16 +42,22 @@ export class DebugView {
         const runDebugButtonLocator: By = By.xpath('//span[@title=\'Start Debugging\']');
 
         await this.driverHelper.waitAndClick(runDebugButtonLocator);
+    }
 
+    /**
+     * Waits for number of threads  in "Threads" view to be more than 1 - this should mean that the debugger is connected.
+     *
+     * @param timeout
+     */
+    async waitForDebuggerToConnect(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
         await this.driverHelper.getDriver().wait(async () => {
             Logger.debug(`Waiting for debugger to connect (threads to appear in "Threads" view)`);
             const threadElements: WebElement[] = await this.driverHelper.getDriver().findElements(By.xpath(`//div[contains(@class, 'theia-debug-thread')]`));
-            if (threadElements.length < 2) {
-                return false;
-            } else {
+            if (threadElements.length > 1) {
                 return true;
             }
-        });
+            return false;
```

Comment:
May be removed, if the condition ```if (threadElements.length > 1)``` will not be satisfied the wait will be failed by timeout.

Reply:
I've just put this there for better readibility.
Could we agree this is very minor and doesn't block merging this PR?